adobe/react-spectrum
# Fix mismatched height between ListView items

ListView items currently have mismatched heights when an ActionGroup is included:

<img width="350" alt="with ActionGroup" src="https://user-images.githubusercontent.com/8961049/168670215-a1d56b36-63ea-493a-a5de-506aae09915b.png">

<img width="350" alt="with ActionButton" src="https://user-images.githubusercontent.com/8961049/168670205-e11b73d6-0d98-4922-bc37-64ac6a901767.png">

This was due to the `overflow: hidden` added to the description styles when adding wrapping/truncation support.

Adding a fixed height to the description fixes this issue. This PR adds a fixed height that matches the listview item's line height (OK for default `overflowMode: truncate`), and reverts to `height: auto` for `overflowMode: wrap`.

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

In ListView stories:
1. Check that item heights match between ActionButton, ActionGroup, and ActionMenu stories
2. Check that overflowMode behavior still works in those stories

Note: Test across browsers, original issue was only present in some.

## 🧢 Your Project:
